contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -3435,3 +3510,415 @@ objectproperty_internal(PG_FUNCTION_ARGS)
 
 	PG_RETURN_NULL();
 }
+
+int 
+SPI_execute_raw_parsetree(RawStmt *parsetree, bool read_only, long tcount)
```

Comment:
Pls add comment for this function illustrate why we have to add this function ( Since we have a discussion about why other SPI_execute not working)